### PR TITLE
Add default with comments for pipe log

### PR DESCRIPTION
**Pull request by paulcsmith**
Part of: https://github.com/luckyframework/lucky/pull/1062

### CODE REVIEW

File: src/web_app_skeleton/config/logger.cr [hunk 1]
```diff
@@ -31,6 +31,15 @@ logger =
...
   settings.logger = logger
+  # By default Lucky only logs when before/after pipes halt (by redirecting, or
+  # rendering a response).
+  #
+  # Set the log level to log to log every pipe that runs.
+  #
+  # For example:
+  #
+  #    settings.action_pipe_log_level = Logger::Severity::DEBUG
+  settings.action_pipe_log_level = nil
```
Comment (paulcsmith) [inline src/web_app_skeleton/config/logger.cr]: Will fail until https://github.com/luckyframework/lucky/pull/1062 is merged.

**Comment by paulcsmith** [comment]
@edwardloveall @jwoertink this will fail until the other PR is merged, but could you take a look at the comment and see if it makes sense?

File: src/web_app_skeleton/config/logger.cr (hunk 1 above)
Comment (edwardloveall) [reply src/web_app_skeleton/config/logger.cr]: Can we also list the other values that `action_pipe_log_level` can be?

File: src/web_app_skeleton/config/logger.cr [hunk 2]
```diff
@@ -31,6 +31,15 @@ logger =
 
 Lucky.configure do |settings|
   settings.logger = logger
+  # By default Lucky only logs when before/after pipes halt (by redirecting, or
+  # rendering a response).
+  #
+  # Set the log level to log to log every pipe that runs.
```
Comment (jwoertink) [inline src/web_app_skeleton/config/logger.cr]:
```suggestion
  # Set the log level to log every pipe that runs.
```

Comment (paulcsmith) [reply src/web_app_skeleton/config/logger.cr]: Oops :)

File: src/web_app_skeleton/config/logger.cr (hunk 1 above)
Comment (paulcsmith) [reply src/web_app_skeleton/config/logger.cr]: There are a bunch but I can link to the Logger docs that show all the possible enums!

**Comment by paulcsmith** [comment]
@jwoertink if it is nil it means they will only log if they are halted by returning a redirect, json, html, etc.

Maybe we should add a resuce as well that logs which pipes have an error, but I'm not sure since that could get hairy and could be caught from the stacktrace

**Comment by paulcsmith** [comment]
I think I addressed everything. Still waiting on the other PR to be green and then I can re-run this on CI

**Pull request merged**